README.md

```diff
@@ -66,5 +66,10 @@ Join the conversation on our [Discord community](https://discord.com/invite/jZQs
       <img src="https://user-images.githubusercontent.com/624760/88266811-c1311900-ccc7-11ea-8863-efa1664b45fd.png" />
       <p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/22">(:100: give your vote)</a></p>
     </td>
+    <td align="center">
+      <p><a href="https://github.com/sandip15">Sandip Roy</a></p>
+      <img src="https://raw.githubusercontent.com/sandip15/sandip15/master/Screen%20Shot%202020-07-25%20at%205.18.19%20PM.png" />
+      <p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/23">(:100: give your vote)</a></p>
```

Comment:
This is not your issue, it is someone else's.

I should probably put some steps in the readme.
1. create an issue with the title as your name and in the description drop your screenshot
2. use the url created by GitHub for the screenshot for the image here
3. use your issue link